requirements-test.txt

```diff
@@ -1,3 +1,3 @@
 docker~=5.0.3
 selenium~=4.1.0
-openwisp-utils[qa]~=1.0.0
+openwisp-utils[qa] @ https://github.com/openwisp/openwisp-utils/tarball/master
```

Comment:
Why is [qa] in this file.
We should not be needing qa in the first place, looks like a recent change! :-O

Reply:
Please @codesankalp when you need to fix failing builds on master, send a separate PR, I need this fix for the other PR too and I am pushing it to master shortly. It's duplicated work from different people.

Reply:
Please remove this and rebase on master: https://github.com/openwisp/docker-openwisp/commit/9acc2001700cb9347d66ada65dedb5564901b3ab.